llvm/test/MC/AVR/inst-brhc.s

```diff
@@ -0,0 +1,24 @@
+; RUN: llvm-mc -triple avr -show-encoding < %s | FileCheck %s
+;
+; RUN: llvm-mc -filetype=obj -triple avr < %s \
+; RUN:     | llvm-objdump -d - \
+; RUN:     | FileCheck --check-prefix=INST %s
+
+foo:
+  brhc .+12
```

Comment:
Having so many instructions could be a maintenance problem.

For other targets we typically pack tests of the same instruction format.

llvm/test/MC/M68k/ and llvm/test/MC/LoongArch/ are good examples.

Reply:
Ok, thanks - I'll take a look!